tests/test_all_fields.py

```diff
@@ -34,8 +34,6 @@
     'payouts':set(),
     'charges': set(),
     'subscription_items':{
-        'tax_rates',
-        'price',
         'updated',
```

Comment:
`tax_rates` and `price` - both fields added to schema. Now it replicates in actual records also. So, removed it from the `KNOWN_MISSING_FIELDS` map.

Reply:
@namrata270998 Please update the PR code changes as per the changes incorporated in 'invoice_line_items' changes - https://github.com/singer-io/tap-stripe/pull/112/files#diff-eeff454edf0e2c75012eb431c28a488069dff76be419b0a1396fc2b451c4d156L51

Reply:
Incorporated the changes as suggested